src/org/zaproxy/zap/extension/ascanrulesBeta/SQLInjectionMySQL.java

```diff
@@ -196,6 +197,10 @@ public void scan(HttpMessage originalMessage, String paramName, String originalP
 			catch (java.net.SocketTimeoutException e) {
 				//to be expected occasionally, if the base query was one that contains some parameters exploiting time based SQL injection?
 				if ( this.debugEnabled ) log.debug("The Base Time Check timed out on ["+msgTimeBaseline.getRequestHeader().getMethod()+"] URL ["+msgTimeBaseline.getRequestHeader().getURI().getURI()+"]");
+			} catch (SocketException ex) {
+				log.info("Caught " + ex.getClass().getName() + " " + ex.getMessage() + 
```

Comment:
I think INFO logging should be used only for occasional 'significant' events, so vote for these sort of events should be DEBUG. Recording stats on them is another matter - that would be really useful, but out of scope for this change.